packages/editor-ui/src/components/ExpressionInput.vue

```diff
@@ -151,7 +151,7 @@ export default mixins(
 
 				let value;
 				try {
-					value = this.resolveExpression(`=${variableName}`);
+					value = this.resolveExpression(`=${variableName.replace(/=&gt;/g, '=>')}`);
```

Comment:
Natively, the editor triggers six renders. The first three with an encoded angle bracket, which causes the expression to fail to resolve and so to be marked red as invalid. The next three use the unencoded angle bracket and so the expression resolves, but the `.invalid` class remains applied, hence red expression with green result.

Adjusting this line only as a workaround to ensure all renders always use the unencoded angle bracket, so that arrow function syntax is not wrongly marked invalid. Ideally we should look into why the editor needs to render so many times, but we are revamping the editor soon so I think did not dig deeper. Let me know otherwise.

![image](https://user-images.githubusercontent.com/44588767/187848654-1386340a-c078-40ad-9115-793e4737f3f0.png)